### PR TITLE
LPS-150434 Update exception message to make it clearer

### PR DESCRIPTION
**Pull request by mateussgn**
Related Issue: https://issues.liferay.com/browse/LPS-150434

Related issue: https://issues.liferay.com/browse/LPS-150434

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -10364,6 +10364,7 @@ object-entry-value-exceeds-integer-field-allowed-size=Object entry value exceeds
 object-entry-value-exceeds-long-field-allowed-size=Object entry value exceeds long field allowed size.
 object-entry-value-exceeds-maximum-long-field-allowed-size=Object entry value exceeds maximum long field allowed size.
 object-entry-value-exceeds-minimum-long-field-allowed-size=Object entry value exceeds minimum long field allowed size.
+object-entry-value-falls-bellow-minimum-long-field-allowed-size=Object entry value falls bellow minimum long field allowed size.
```
Comment (marcelabc) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:10367]: I have one doubt: you added one language key, but don't we need to delete the other one that you replaced?

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hey @mateussgn, how are you doing?
I finished your review! Let me know if there is any question.

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties (hunk 1 above)
Comment (mateussgn) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:10367]: I forgot to delete the old one, doing it right now.

**Comment by mateussgn** [comment]
ci:stop

**Comment by liferay-forms** [comment]
Hey @mateussgn, can you please check if these failures are related to your changes? Thanks

**Comment by mateussgn** [comment]
ci:test:relevant

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:forward

**Comment by liferay-forms** [comment]
ci:test:relevant

**Pull request closed without merging**